Authenticator/UI/AccountRowView.swift

```diff
@@ -126,7 +126,7 @@ struct AccountRowView: View {
                         .stroke(pillColor, lineWidth: 1)
                 }
                 .accessibilityElement()
-                .accessibilityLabel(account.state == .expired ? "Code expired" : account.formattedCode ?? "Code not calculated")
+                .accessibilityLabel(account.state == .expired ? String(localized: "Code expired", comment: "Accessibility label") : account.formattedCode ?? String(localized: "Code not calculated", comment: "Accessibility label"))
```

Comment:
should the comments be something different than "Accessibility label" ?

Reply:
I understand now that the comment is for the use of the localized string and it comments that it is used in accessibilityLabel. I am resolving the conversation.